shared/torngit/github.py

```diff
@@ -609,6 +610,33 @@ def count_and_get_url_template(cls, url_name):
         GITHUB_API_ENDPOINTS[url_name]["counter"].inc()
         return GITHUB_API_ENDPOINTS[url_name]["url_template"]
 
+    async def build_comment_request_body(
+        self, body: dict, issueid: int | None = None
+    ) -> dict:
+        body = dict(body=body)
+        try:
+            ownerid = self.data["owner"].get("ownerid")
+            if (
+                issueid is not None
+                and await INCLUDE_GITHUB_COMMENT_ACTIONS_BY_OWNER.check_value_async(
+                    identifier=ownerid, default=False
+                )
+            ):
+                bot_name = get_config(
+                    "github", "comment_action_bot_name", default="sentry-ai"
+                )
+                body["actions"] = [
+                    {
+                        "name": "Open Sentry Agent",
+                        "type": "copilot-chat",
+                        "prompt": f"@{bot_name} generate tests for PR",
+                    }
+                ]
+        except Exception:
```

Comment:
for a try-catch that's this broad, it would be a good idea to log the exceptions with a distinct name, so you can monitor and look up whether any exceptions are happening here